heron/common/src/java/com/twitter/heron/common/test/CommunicatorTestHelper.java

```diff
@@ -11,14 +11,16 @@
 //  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 //  See the License for the specific language governing permissions and
 //  limitations under the License.
-package com.twitter.heron.common.basics;
+package com.twitter.heron.common.test;
```

Comment:
Can you make the package name as com.twitter.heron.common.testhelpers or something similar - you don't want to give the impression that it is some kind of test.